### PR TITLE
[FIX] account: cash rounding precision

### PR DESCRIPTION
**Pull request by nim-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/models/account_cash_rounding.py [hunk 1]
```diff
@@ -29,8 +29,8 @@ class AccountCashRounding(models.Model):
     @api.constrains('rounding')
     def validate_rounding(self):
         for record in self:
-            if record.rounding < 0:
-                raise ValidationError(_("Please set a positive rounding value."))
+            if record.rounding <= 0:
+                raise ValidationError(_("Please set a strictly positive rounding value."))
```
Comment (oco-odoo) [inline addons/account/models/account_cash_rounding.py:33]: Shouldn't you update the .pot as well, then?

**Review by oco-odoo — COMMENTED** [review]
Just a small remark, otherwise I'm okay with it.

File: addons/account/models/account_cash_rounding.py (hunk 1 above)
Comment (nim-odoo) [reply addons/account/models/account_cash_rounding.py:33]: Damn, I was expecting you'd miss that :D

**Comment by nim-odoo** [comment]
robodoo r+

**Pull request closed without merging**